cmd/plural/deploy.go

```diff
@@ -247,7 +250,25 @@ func (p *Plural) deploy(c *cli.Context) error {
 				fmt.Println("")
 			}
 		}
-
+		for _, ch := range man.Charts {
```

Comment:
We should send this info from the server if we want it in posthog, having the cli send it is error prone and could potentially cause failed installs

Reply:
We can't track the actual commands users execute server side since it doesn't know when somebody runs `plural deploy`. Having it in the CLI allows us to have a better metric of which commands people run, how often they run, and if they encountered an error when running the command. This can give us a metric of how many successful deploys have happened versus how many errors have occurred. It can also give us a timeline of a user's actions. For example, user deploys but runs into an error for app X, user then runs `install` to change a config and `build` for app X to fix that config, user then runs `deploy` again and it runs successfully.

Each command that a user executes and if it passes or fails is very important data so we can have a better idea of how many command executions pass versus fail and where they fail exactly.